### DISCUSSION TITLE
LLM Controller Interface for vLLM (LLM output format controller)

### DISCUSSION THREAD

**Discussion by Ksuriuri** [vllm-project/vllm#4578] [Show and tell]
I made a simple implementation of controllable generation of large language models based on vLLM, enabling precise control over the model's output format through additional control information: [Ksuriuri/LLMCI](https://github.com/Ksuriuri/LLMCI)

The performance looks good to me, feel free to try it if you need😊

**Comment by AaronFriel** [comment]
Neat project and clever use of stop tokens to add fixed content! That said, this isn't quite the same as AICI, so I wonder if it should share the same name?

**Reply by Ksuriuri** [reply]
Thank you for your comment and suggestion! I think LLMCI is more suitable and I change the name.
